_includes/devdoc/dash-core/rpcs/rpcs/gettransaction.md

```diff
@@ -52,11 +52,6 @@ The `gettransaction` RPC {{summary_getTransaction}}
   p: "Optional<br>(0 or 1)"
   d: "If an outgoing transaction, this is the fee paid by the transaction reported as negative dash"
 
-- n: "→<br>`instantlock`"
-  t: "bool"
-  p: "Required<br>(exactly 1)"
-  d: "Current transaction lock state"
```

Comment:
Why's this being removed here?

Reply:
It's now included in the `{{INCLUDE_F_LIST_TRANSACTIONS_F_FULL}}` reference in the next line (which is defined in `_includes/helpers/vars.md`).